generic.js

```diff
@@ -2,4 +2,6 @@
 
 if(vehicle === "motorbike") {
   console.log("You should add a couple of wheels")
+} else if(vehicle ===    "car") {
+  console.log('Life is good')
```

Comment:
Codacy has a fix for the issue: [Replace `'Life·is·good')` with `"Life·is·good");`](https://release.k8s.dev.codacy.org/gh/machadoit-test-org/eslint-example/pullRequest?prid=5799)
```suggestion
  console.log("Life is good");
```